src/joinboost/app.py

```diff
@@ -128,20 +128,25 @@ def _build_model(self):
             self.model_def.append(cur_model_def)
         
     def compute_rmse(self, test_table: str):
+        if self.cjt.is_target_relation_a_view():
```

Comment:
TODO: 

- We will store the relation-view mapping in the app rather than the cjt